engine/access/rest/scripts_test.go

```diff
@@ -39,8 +39,8 @@ func scriptReq(id string, height string, body interface{}) *http.Request {
 }
 
 func TestScripts(t *testing.T) {
-	validCode := []byte(`pub fun main(foo: String): String { return foo }`)
-	validArgs := []byte(`{ "type": "String", "value": "hello world" }`)
+	validCode := []byte(`pub fun main(foo: Name): Name { return foo }`)
+	validArgs := []byte(`{ "type": "Name", "value": "hello world" }`)
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig